### ISSUE TITLE
Class features granted by feats are deleted when lowering PC level

### ISSUE THREAD

**Issue by GorgoPrimus** [foundryvtt/pf2e#9598]
![Untitled](https://github.com/foundryvtt/pf2e/assets/126842722/047a8951-eae2-4b05-b8f5-273fd441aa9d)
As shown here on this Monk with the Barbarian Dedication (Spirit), the -1 AC from Rage isn't being taken into account, and it appears the +2 damage isn't either.

**Comment by GorgoPrimus** [comment]
I figured out what caused this. If you go from level 0 to 5 to re-fetch class feats, it won't re-add your Dedication Archetype Feats back properly and so those need to be deleted and readded (like it'll add Barbarian Dedication, but not what kind you are or add Rage; it'll add Sorcerer Dedication but not your bloodline; etc). No idea if this is purely a "so know that and deal with it next time" issue or a "system should probably prompt you to repick these things when you do this, so we'll look into it" issue.

Either way, I thought I should update this with what I learned.
